pkg/service/backup/service_backup_restore_integration_test.go

```diff
@@ -1077,11 +1271,23 @@ func grantPermissionsToUser(s gocqlx.Session, target RestoreTarget, user string)
 	var ks, t string
 	iter := s.Query("SELECT keyspace_name, table_name FROM system_schema.tables", nil).Iter()
 	for iter.Scan(&ks, &t) {
+		// Regular tables require ALTER permission.
 		if f.Check(ks, t) {
 			if err = s.ExecStmt(fmt.Sprintf("GRANT ALTER ON %s.%s TO %s", ks, t, user)); err != nil {
 				return errors.Wrap(err, "grant alter permission")
 			}
 		}
+		// Views of restored base tables require DROP and CREATE permissions.
```

Comment:
Needs to be documented https://manager.docs.scylladb.com/master/restore/restore-tables.html#prerequisites

Reply:
That's true!
Generally there are more thing to change in restore docs because of this PR, but I was going to wait with that until we agree that this implementation is what we are wanting to go through with.